CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+Release v0.15.0 (2019-10-18)
+===
+
+### Services
```

Comment:
Not we're inconsistent with the previous releases header format `###` vs `---`. The latter is more convenient because it's compatible with got commit message.